pkgs/applications/networking/instant-messengers/pantalaimon/default.nix

```diff
@@ -9,13 +9,16 @@
 
 buildPythonApplication rec {
   pname = "pantalaimon";
-  version = "0.4";
+  version = "0.5.1";
 
   disabled = pythonOlder "3.6";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "1canj9w72wh1rcw6fivrvaahpxy13gb6gh1k8nss6bgixalvnq9m";
+  # pypi tarball miss tests
+  src = fetchFromGitHub {
```

Comment:
out of curiosity, is there a reason to use github over pypi?

Reply:
presumably because `pypi tarball miss tests`?

Reply:
yes.

Reply:
heh, of course. derp :)